### PR TITLE
feat(releases): show indexer name instead of identifier

### LINKED ISSUE #1705: [FEATURE REQ] Display user defined indexer names on dashboard and releases page
### Discussed in https://github.com/autobrr/autobrr/discussions/1698

<div type='discussions-op-text'>

<sup>Originally posted by **onedr0p** September  1, 2024</sup>
**Is your feature request related to a problem? Please describe.**

Currently the indexer names on these pages are pulled from the [identifier in the indexer definition](https://github.com/autobrr/autobrr/blob/34d6e0cf609e225fa22261177d4b43c258f03566/internal/indexer/definitions/hd-torrents.yaml#L4)

**Describe the solution you'd like**

I would like for the Dashboard and Releases page to display the indexer names I've set in the indexer settings.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A
</div>

### PR DESCRIPTION
**Pull request by zze0s**
Show Indexer Name in releases list instead of `identifier`.

Fixes #1705

### CODE REVIEW

**Comment by martylukyy** [comment]
resolves https://github.com/autobrr/autobrr/discussions/1698

File: web/src/components/data-table/Cells.tsx [hunk 1]
```diff
@@ -110,23 +110,23 @@ export const AgeCell = ({value}: CellProps<Release>) => (
...
-    className={classNames(
-      "py-3 text-sm font-medium box-content text-gray-900 dark:text-gray-300",
-      "max-w-[96px] sm:max-w-[216px] md:max-w-[360px] lg:max-w-[640px] xl:max-w-[840px]"
-    )}
-  >
-    <Tooltip
-      requiresClick
-      label={value}
-      maxWidth="max-w-[90vw]"
+export const IndexerCell = (props: CellProps<Release>) => (
```
[the remark this answers is not in the document]
Comment (zze0s) [reply web/src/components/data-table/Cells.tsx:113]: Release.Indexer already has the 'MinimalIndexer` type.

**Comment by martylukyy** [comment]
Speaking of removing unused code...

During review i noticed that there is a lot of unused code in ActivityTable.tsx... 🤔 

https://github.com/autobrr/autobrr/blob/8d922caee0bd2004374b0cd8a1cbf08f01aaaca1/web/src/screens/dashboard/ActivityTable.tsx#L170-L190
https://github.com/autobrr/autobrr/blob/8d922caee0bd2004374b0cd8a1cbf08f01aaaca1/web/src/screens/dashboard/ActivityTable.tsx#L280-L355

If i read this correctly, this code isn't currently used since the router does the loading animation of the Dashboard
until the data for the table is fetched. The stats will then pulsate until it's data is available to show...
Should we get rid of this?

**Comment by zze0s** [comment]
Done @martylukyy

**Review by martylukyy — APPROVED** [review]
Looks good to me! Nice cleanup aswell! 🚀

**Comment by onedr0p** [comment]
thanks @zze0s this helps with UX 👍

**Pull request merged**